src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/EnvelopeHandler.java

```diff
@@ -44,13 +44,15 @@ public void handleEnvelope(Envelope envelope) {
                         paymentsProcessor.createPayments(envelope, existingCase.getId(), false);
                     } else {
                         log.info(
-                            "Creating exception record as supplementary evidence failed for envelope {} case {}",
+                            "Adding supplementary evidence from envelope {} to case {} failed. "
+                                + "Creating exception record instead.",
```

Comment:
took me a while to realise 'Creating exception record as supplementary evidence failed' meant 'because', not 'creating X as something Y' failed.

Reply:
Message change seems right. Maybe we should log it as an exception? This is an error situation, that we should be alerted about, investigate and most likely product team will have to fix their definition as a result.

Reply:
Yes that's correct we need to log it as an exception otherwise it will too late for us to figure out something has gone wrong. On another note we tend to log everything as error just so that we can get error email. Ideally we should create different alert config in app insights and not just make it log as an error.

Reply:
Worth exploring that possibility

Reply:
it's being logged as an error in https://github.com/hmcts/bulk-scan-orchestrator/blob/39dccc83d766d859814da2c2b5f65061a3d2bcd6/src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/AttachDocsToSupplementaryEvidence.java#L89